yapapi/executor/__init__.py

```diff
@@ -415,13 +403,11 @@ async def respond_to_proposal(proposal, builder):
                             type(strategy).__name__,
                             score,
                         )
-                    except InvalidPropertiesError as err:
-                        await reject_proposal(proposal, "Malformed offer")
-                        continue
-                    if score < SCORE_NEUTRAL:
-                        await reject_proposal(proposal, "Score too low")
-                    elif not proposal.is_draft:
-                        try:
```

Comment:
This error will be handled by the `except Exception` clause at the end of `find_offers()`.